packages/indexer/src/sinks/drizzle/drizzle.test.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  type MockBlock,
+  MockClient,
+  type MockFilter,
+} from "@apibara/protocol/testing";
+import { eq, sql } from "drizzle-orm";
+import { drizzle } from "drizzle-orm/node-postgres";
+import { serial, text } from "drizzle-orm/pg-core";
+import { Client } from "pg";
+import { beforeAll, beforeEach, describe, expect, it } from "vitest";
+import { type Int8Range, drizzle as drizzleSink, pgTable } from ".";
+import { useSink } from "../../hooks";
+import { run } from "../../indexer";
+import { generateMockMessages } from "../../testing";
+import { getMockIndexer } from "../../testing/indexer";
+
+const test_table = pgTable("test_table", {
```

Comment:
```suggestion
const testTable = pgTable("test_table", {
```

I make the same mistake all the time. The name of the table should be snake case, but the variable can follow typsecript naming standards.